ccd-definition/CaseField/CareSupervision/caseField.json

```diff
@@ -89,6 +89,15 @@
     "SecurityClassification": "Public",
     "Searchable": "N"
   },
+  {
+    "LiveFrom": "01/01/2018",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "ID": "taskListWelsh",
+    "Label": "Case name",
```

Comment:
Does this label need to be in Welsh

Reply:
Shouldn't be necessary, this is the hidden field which is then embedded in a different field for markdown rendering, so this won't be displayed, not even sure why it was labeled as "case name" in english either though 😆